src/chain/chain.rs

```diff
@@ -481,6 +470,21 @@ impl Chain {
             .should_vote_for_merge(self.elder_size(), self.neighbour_infos())
     }
 
+    /// Increase version by one due to the parsec pruning.
```

Comment:
Sorry, I'm not sure why we increase the ElderInfo version, this should be an internal thing only, shouldn't it?

Reply:
The parsec_map creates parsec instance based on version which is deduced from the ElderInfo version.
If we don't create a new ElderInfo (with version increased by one here), there is no new parsec instance will got created.

Reply:
Yes, it seem to be the case.
This is a short cut that is no longer reasonable in the context of Parsec Prune, that value should probably be in GenesisPfxInfo, and incremented independently of SectionInfo.

(We do not want to notify our neighbour that things have changed just because we locally are gossiping to much data. (It would make us gossip even more data which is not ideal in a prune scenario).

Reply:
I was just want to limit the affects to the existing structs & approaches.
I agree with the concern of notify neighbours.
However, without a version increased, the parsec_map has to `replace` the instance, which could cause other issues? (as there are many other stuff depends on `version` I think? )